### DISCUSSION TITLE
Idea: Support for shap-hypetune library (Boruta, RFS, and/or RFA) for automatic feature selection

### DISCUSSION THREAD

**Discussion by tsereno** [microsoft/FLAML#499] [Ideas]
I have observed empirical improvements to model training metrics when using BorutaShap and shap-hypetune libraries.  Just sharing this idea that including automatic feature selection capability to FLAML could add value to the autoML pipeline with potential outcome of improved model metrics.

**Comment by sonichi** [comment]
Thanks for the suggestion. Would you please share some concrete examples of the observed empirical improvements? Code examples would be most helpful.
